test/libyul/YulInterpreterTest.cpp

```diff
@@ -87,8 +87,8 @@ bool YulInterpreterTest::parse(ostream& _stream, string const& _linePrefix, bool
 string YulInterpreterTest::interpret()
 {
 	InterpreterState state;
-	state.maxTraceSize = 10000;
-	state.maxSteps = 10000;
+	state.maxTraceSize = 10;
+	state.maxSteps = 150;
```

Comment:
Can we increase these?

I'd suggest something like `maxTraceSize = 32` and `maxSteps = 512`.

Reply:
Done

Reply:
Did you push?

Reply:
Ah sorry, just pushed!